pkg/parsers/kernel/kernel.go

```diff
@@ -45,6 +47,26 @@ func CompareKernelVersion(a, b VersionInfo) int {
 	return 0
 }
 
+// CompareKernelVersionNproc checks if current is 4.10+ to check nproc feature
+func CompareKernelVersionNproc(a VersionInfo) int {
+	if a.Major < 4 && a.Minor < 10 {
+		return 1
+	}
+	return 0
+}
+
+// CheckKernelVersionNproc checks if current kernel is 4.10+.
+func CheckKernelVersionNproc() bool {
+	if v, err := GetKernelVersion(); err != nil {
+		logrus.Warnf("error getting kernel version: %s", err)
+	} else {
+		if CompareKernelVersionNproc(*v) != 0 {
```

Comment:
Could you combine the else and if using 
```
else if CompareKernelVersionNproc(*v) != 0{
         return false
}
```
I think this could be more readable.